Makefile

```diff
@@ -0,0 +1,37 @@
+
+BIN:=iso-chooser-menu
+
+URL:=http://cdimage.ubuntu.com/streams/v1/com.ubuntu.cdimage.daily:ubuntu-server.json
+
+CFLAGS:=-Wall -Werror -Wfatal-errors
+LDFLAGS:=-static
```

Comment:
We don't statically link most executables in the initrd aiui but rather rely on copy_exec to copy the dependencies in. It doesn't look as if libncurses or libjson-c are in the initrd already but it might be interesting to compare the difference between including them vs including the static binary.

Reply:
On early analysis, the copy_exec magic with a non-static binary does appear to be smaller (difference of 660kiB), will look further on how that works and likely switch to the dynamic binary.

Reply:
Final number was 550kiB, still worth doing.
Implemented.